jobbergate-docs/src/conf.py

```diff
@@ -30,11 +30,17 @@
 pygments_style = "rainbow_dash"
 exclude_patterns = []
 extensions = [
-    "sphinx.ext.githubpages",
+    "sphinx.ext.autodoc",
     "sphinx.ext.autosectionlabel",
+    "sphinx.ext.githubpages",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
     "sphinxcontrib.httpdomain",
 ]
```

Comment:
a huge addition here IMO is `sphinx_copybutton`. The code snippets can be coppied to clipboard quite easily. Check the links bellow for details:

[1] https://github.com/omnivector-solutions/armada-infrastructure/blob/main/docs-source/conf.py#L41
[2] https://github.com/omnivector-solutions/armada-infrastructure/blob/main/pyproject.toml#L17

Reply:
Totally